package.json

```diff
@@ -69,18 +69,18 @@
     "gulp-uglify": "^3.0.2",
     "html-validate": "7.5.0",
     "html5shiv": "^3.7.3",
-    "jest": "^29.2.1",
-    "jest-axe": "^6.0.0",
+    "jest": "^29.3.1",
+    "jest-axe": "^7.0.0",
     "jest-dev-server": "^6.1.1",
-    "jest-environment-jsdom": "^29.2.1",
+    "jest-environment-jsdom": "^29.3.1",
     "jest-environment-puppeteer": "^6.1.1",
     "jest-serializer-html": "^7.1.0",
     "jquery": "1.12.4",
     "js-yaml": "^4.1.0",
-    "jsdoc": "^3.6.11",
+    "jsdoc": "^4.0.0",
```

Comment:
Breaking change because they've dropped the [`taffydb`](https://www.npmjs.com/package/taffydb) package (which we don't use)